packages/ui/primitives/document-dropzone.tsx

```diff
@@ -104,11 +104,12 @@ export const DocumentDropzone = ({
 
   return (
     <motion.div
-      className={cn('flex', className)}
+      className={cn('flex aria-disabled:cursor-none', className)}
```

Comment:
`cursor-not-allowed` would be better I think, good use of aria-disabled though I was going to ask for it but forgot to